### PR TITLE
Sprint3/on 59

### PR DESCRIPTION
**Pull request by 22jen**
Done with scraping_articles code. Removed __init__.py only for "get_article" file because it was unnecessary.

### CODE REVIEW

**Review by pinkcoderpro718 — APPROVED** [review]
This seems good
Good job

File: readme [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# import requests
```
Comment (Mohan-sharma-Venkatesh) [inline readme:1]: I think read me is updated by mistake

Comment (Mohan-sharma-Venkatesh) [reply readme:1]: please do remove it

File: news_sentiment_analysis/scrape_articles/get_article_urls/__init__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# __init__.py
+import serpapi
+import pandas as pd
+
+API_KEY = '6ff7904b85915dcc6a2edb207e7b3c377faa56ef237c66cf6d87ba30eb123226'
+
+def get_default_params(query, country_code):
```
Comment (Mohan-sharma-Venkatesh) [inline news_sentiment_analysis/scrape_articles/get_article_urls/__init__.py:7]: Can you move get_default_params()
from __init__.py -> bing_news.py

Comment (Mohan-sharma-Venkatesh) [reply news_sentiment_analysis/scrape_articles/get_article_urls/__init__.py:7]: __init__.py contains only import of subpackages and packages

Comment (Mohan-sharma-Venkatesh) [reply news_sentiment_analysis/scrape_articles/get_article_urls/__init__.py:7]: https://stackoverflow.com/questions/448271/what-is-init-py-for#:~:text=Files%20named%20__init__.py%20are%20used%20to%20mark%20directories%20on%20disk%20as%20Python%20package%20directories.%20If%20you%20have%20the%20files

Please do look at this

**Review by pinkcoderpro718 — APPROVED** [review]
Clean and tidy!

**Pull request merged**